modules/compute-vm/variables.tf

```diff
@@ -35,7 +35,7 @@ variable "attached_disk_defaults" {
 }
 
 variable "attached_disks" {
-  description = "Additional disks, if options is null defaults will be used in its place. Source type is one of 'image' (zonal disks in vms and template), 'snapshot' (vm), 'existing', and null."
+  description = "Additional disks, if options is null defaults will be used in its place. Source type is one of 'image' (zonal disks in vms and template), 'snapshot' (vm), 'attach', and null."
```

Comment:
attach is used to flag a disk needs to be created and managed by the module

![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/66007/27b64224-aa4d-4638-a747-429775eb57aa)

Reply:
Yes, but you cannot use "existing" as value, but only one between 'image', 'snapshot', 'attach', 'null'.